persistence-cassandra/core/src/main/scala/com/lightbend/lagom/internal/persistence/cassandra/CassandraReadSideSettings.scala

```diff
@@ -14,4 +15,5 @@ private[lagom] class CassandraReadSideSettings @Inject() (system: ActorSystem) {
   private val cassandraConfig = system.settings.config.getConfig("lagom.persistence.read-side.cassandra")
 
   val autoCreateTables: Boolean = cassandraConfig.getBoolean("tables-autocreate")
+  val writeConsistency: ConsistencyLevel = ConsistencyLevel.valueOf(cassandraConfig.getString("write-consistency"))
```

Comment:
I think the lagom `CassandraSession` has a `CassandraSessionSettings` which should already contain the `writeConsistency`. If there was access to that `cassandraSessionSettings` from the `CassandraReadSideHandler` you're changing below it may be cleaner since we're reading each setting only once.

I think the goal of `CassandraReadSideSettings` was to only provide access to lagom-specific settings.

Reply:
That would simplify this PR since most changes are passing those settings around, I'll take a look.

Reply:
Nope, that is not available as a field on the `CassandraSession` and even more tricky, what is passed around is in some places rather `CassandraSession` from APC, so I think just adding this here is less invasive/risky a change even if perhaps not the most idiomatic solution.